mne/viz/evoked.py

```diff
@@ -477,7 +477,7 @@ def whitened_gfp(x, rank=None):
 
     if not has_sss:
         evokeds_white[0].plot(unit=False, axes=axes_evoked,
-                              hline=[-1.96, 1.96])
+                              hline=[-1.96, 1.96], show=False)
```

Comment:
in other words, the show gets delayed until the end of the function, which I think is the right behavior, no?